src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectRequestType.cs

```diff
@@ -34,9 +34,9 @@ namespace Microsoft.IdentityModel.Protocols.OpenIdConnect
     public enum OpenIdConnectRequestType
     {
 #pragma warning disable 1591
```

Comment:
Can we pull out this disabling of the compiler warning and correctly xml-doc this member?